tests/snapshot/snapshot/snapshot.yaml

```diff
@@ -6,21 +6,27 @@ epilogues:
     method: DELETE
     status: [200, 404]
     parameters:
-      repository: my-fs-repository
```

Comment:
We have a bunch of tests where we say `my-fs-repository`, either change them all or change this back to `my-fs-repository`.